CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+## HEAD
+- Alias `acts_as_taggable_on` to `taggable_on`
```

Comment:
There seems to be a mismatch with actual implementation. You probably meant `taggable_array_on`.